.gitignore

```diff
@@ -2,6 +2,7 @@
 
 /tmp/
 docs/_build
+docs/lint-checks.rst
```

Comment:
We should probably add this file to the `clean` target of `Makefile` well.

Reply:
Ah, there's `Makefile` under `docs` as well ;-) I was thinking about the one in the repo root. This would work nice as well but we need to call `clean` from the main `Makefile`.

Reply:
Maybe this then?

```
diff --git a/Makefile b/Makefile
index 69696666..616ce660 100644
--- a/Makefile
+++ b/Makefile
@@ -92,3 +92,4 @@ clean:
        rm -f .coverage tags
        rm -rf examples/convert/{main.fmf,test.md,Manual}
        rm -f tests/full/repo_copy.tgz
+       make -C docs/ clean
```